single-template.php

```diff
@@ -223,7 +223,15 @@ function dt_display_tile( $tile, $post ): bool {
                             ?>
                             </span>
                             </dt-modal>
-                            <div class="details-title-section"></div>
+                            <!-- duplicate check button integration -->
+                            <dt-modal class="duplicate-detected" buttonClass={"duplicates-detected-button":true} buttonLabel="Duplicate Detected" buttonStyle={"background-color":"transparent","border":"1px","color":"#3f729b","border-style":"solid","font-weight":"600","border-color":"#3f729b","display":""} imageStyle={"filter":"hue-rotate(0deg)"} imageSrc="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/broken.svg' ) ?>" title="Duplicate Detected">
```

Comment:
@micahmills curious if all buttons need this much css? Can there be a default?
And translation for the title?

Reply:
hey @corsacca, it shouldn't need CSS, we need to upgrade the dt-modal component to use the dt-button component. I also missed the translation not being translated. I will take care of that now.